lib/node_modules/@stdlib/math/generics/statistics/incrprod/README.md

```diff
@@ -0,0 +1,123 @@
+# incrprod
+
+> Compute a product incrementally.
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var incrprod = require( '@stdlib/math/generics/statistics/incrprod' );
+```
+
+#### incrprod()
+
+Returns an accumulator `function` which incrementally computes a product.
+
+``` javascript
+var accumulator = incrprod();
+```
+
+#### accumulator( \[x\] )
+
+If provided an input value `x`, the accumulator function returns an updated product. If not provided an input value `x`, the accumulator function returns the current product.
+
+``` javascript
+var product = accumulator( 2.0 );
+// returns 2.0
+
+product = accumulator( 1.0 );
+// returns 3.0
+
+product = accumulator( 3.0 );
+// returns 6.0
+
+product = accumulator();
+// returns 6.0
+```
+
+</section>
+
+<!-- /.usage -->
+
+
+<section class="notes">
+
+## Notes
+
+* Input values are __not__ type checked. If provided `NaN` or a value which, when used in computations, results in `NaN`, the accumulated value is `NaN` for __all__ future invocations. If non-numeric inputs are possible, you are advised to type check and handle accordingly __before__ passing the value to the accumulator function.
+* For long running accumulations or accumulations of large numbers, care should be taken to prevent overflow. Overflow need not be forever since internally the accumulator stores the product as a __mantissa__ and __exponent__.
```

Comment:
Should update the note regarding overflow to include underflow. e.g., 

> For long running accumulations or accumulations of either large or small numbers, care should be taken to prevent overflow and underflow. Note, however, that overflow/underflow may be transient, as the accumulator does not use a double to store an accumulated product, but instead, updates the normalized __mantissa__ and __exponent__ of a product separately. Doing so guards against a loss in precision.